### PR TITLE
docs(components): [select-v2] use new display tag

### PR DESCRIPTION
**Pull request by wzc520pyfm**
Related: #10754 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/select-v2.md [hunk 1]
```diff
@@ -203,27 +205,27 @@ select-v2/props
...
+| Name           | Description                                                                                                | Type                                |
+| -------------- | ---------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| change         | triggers when the selected value changes, the param is current selected value                              | ^[Function]`(val: any) => void`     |
+| visible-change | triggers when the dropdown appears/disappears, the param will be true when it appears, and false otherwise | ^[Function]`(val: boolean) => void` |
+| remove-tag     | triggers when a tag is removed in multiple mode, the param is removed tag value                            | ^[Function]`(val: any) => void`     |
+| clear          | triggers when the clear icon is clicked in a clearable Select                                              | ^[Function]`() => void`             |
+| blur           | triggers when Input blurs                                                                                  | ^[Function]`(event: FocusEvent)`    |
```
Comment (btea) [inline docs/en-US/component/select-v2.md:225]:
```suggestion
| blur           | triggers when Input blurs                                                                                  | ^[Function]`(event: FocusEvent) => void`    |
```

File: docs/en-US/component/select-v2.md [hunk 2]
```diff
@@ -203,27 +205,27 @@ select-v2/props
...
+| focus          | triggers when Input focuses                                                                                | ^[Function]`(event: FocusEvent)`    |
```
Comment (btea) [inline docs/en-US/component/select-v2.md:226]: The same as above.

File: docs/en-US/component/select-v2.md [hunk 3]
```diff
@@ -203,27 +205,27 @@ select-v2/props
 | loading-text | 远程加载时显示的文字 | string | — | 加载中 | -->
 </span>
 
-## props
+#### props
```
Comment (btea) [inline docs/en-US/component/select-v2.md:208]: Maybe the `props` level just needs to be set to `### props`?

Comment (wzc520pyfm) [reply docs/en-US/component/select-v2.md]: i agree

**Review by kooriookami — APPROVED** [review]
Nice

**Review by btea — APPROVED** [review]

**Pull request merged**